### PR TITLE
Removal of Recipes from Kitchen Migration

### PR DESCRIPTION
**Pull request by fayvor**
[#138](https://github.com/ibm-granite-community/pm/issues/138)
- delete recipes
- pare down dependencies in setup.py
- add notebook utils tests
- rewrite README for Utils
- move README and LICENSE to the top (following pattern used in the other repos)

### CODE REVIEW

**Comment by rawkintrevo** [comment]
per the failed test, you need to install `dotenv` somewhere

File: setup.py [hunk 1]
```diff
@@ -7,16 +7,7 @@
     version='0.1.0',
     packages=find_packages(),
     install_requires=[
-        'python-dotenv',
```
Comment (rawkintrevo) [inline setup.py:10]: this being dropped is my best guess of where the test failure is coming from

**Review by rawkintrevo — CHANGES_REQUESTED** [review]
something is wonk with that dotenv being gone. Maybe try adding `python-dotenv` back in?

File: setup.py (hunk 1 above)
Comment (fayvor) [reply setup.py:10]: Just re-added, thx.

**Comment by rawkintrevo** [comment]
lgtm!

**Pull request merged**